docs/language-usage.rst

```diff
@@ -0,0 +1,354 @@
+Assuming your environment is set up as described in that document,
+create an instance of :class:`~gcloud.language.client.Client`.
+
+  .. code-block:: python
+
+     >>> from gcloud import language
+     >>> client = language.Client()
+
+Methods
+-------
+
+The Google Natural Language API has three supported methods
+
+- `analyzeEntities`_
+- `analyzeSentiment`_
+- `annotateText`_
+
+and each method uses a `Document`_ for representing text. To
+create a :class:`~gcloud.language.document.Document`,
+
+  .. code-block:: python
+
+     >>> text_content = (
+     ...     'Google, headquartered in Mountain View, unveiled the '
+     ...     'new Android phone at the Consumer Electronic Show.  '
+     ...     'Sundar Pichai said in his keynote that users love '
+     ...     'their new Android phones.')
+     >>> document = client.document(text_content)
+
+By default, the document's language and type are English and
+plain text:
+
+  .. code-block:: python
+
+     >>> document.language
+     'en'
+     >>> document.doc_type
+     'PLAIN_TEXT'
+
+These values can also be specified in the constructor
+
+  .. code-block:: python
+
+     >>> html_content = """\
+     ... <html>
+     ...   <head>
+     ...     <title>Story Time</time>
+     ...   </head>
+     ...   <body>
+     ...     <p>La vaca salt&oacute; sobre la luna.</p>
+     ...   </body>
+     ... </html>
+     ... """
+     >>> document = client.document(html_content,
+     ...                            language='es',
+     ...                            doc_type=language.Document.HTML)
+
+The ``language`` arugment can be either ISO-639-1 or BCP-47 language
+codes; at the time, only English, Spanish, and Japanese are supported.
+However, the ``analyzeSentiment`` method only supports English text.
+
+The document type (``doc_type``) argument can be one of
+:attr:`Document.PLAIN_TEXT <gcloud.language.document.Document.PLAIN_TEXT>` or
+:attr:`Document.HTML <gcloud.language.document.Document.HTML>`.
+
+In addition to supplying the text / HTML content, a document can refer
+to content stored in `Google Cloud Storage`_:
+
+  .. code-block:: python
+
+     >>> document = client.document(bucket='my-text-bucket',
+     ...                            blob='sentiment-me.txt')
+     >>> document.gcs_url
+     'gs://my-text-bucket/sentiment-me.txt'
+
+.. _analyzeEntities: https://cloud.google.com/natural-language/reference/rest/v1beta1/documents/analyzeEntities
+.. _analyzeSentiment: https://cloud.google.com/natural-language/reference/rest/v1beta1/documents/analyzeSentiment
+.. _annotateText: https://cloud.google.com/natural-language/reference/rest/v1beta1/documents/annotateText
+.. _Document: https://cloud.google.com/natural-language/reference/rest/v1beta1/Document
+.. _Google Cloud Storage: https://cloud.google.com/storage/
+
+Analyze Entities
+----------------
+
+The :meth:`~gcloud.language.document.Document.analyze_entities` method
+finds named entities (i.e. proper names) in the text, entity types,
```

Comment:
can we put the examples in-line here? particularly for salience which often confuses people.